### DISCUSSION TITLE
macOS版本

### DISCUSSION THREAD

**Discussion by mccation** [Meloong-Git/PCL#2060] [提问] [answered]
本人想买MacBook 请问能做mac版本启动器吗

**Comment by iPodtouch6557** [comment, marked as answer]
见 #54 ，不可以qwq
PCL2 基于 WPF 开发，其架构决定了它无法基于除 Windows 以外的任何操作系统来运行，如果要那么搞等同于完全重做，而那是不可能的……

**Comment by Chiloven945** [comment]
WPF开发环境仅针对windows
不过你可以通过在mac上安装windows的方法来使用启动器
